### DISCUSSION TITLE
Deleting repositories remove badges as well?

### DISCUSSION THREAD

**Discussion by OlaPietka** [Schweinepriester/github-profile-achievements#188] [Q&A] [answered]
Hi there! I was wondering if my badge will disappear after deleting the repository that is linked to that achievement. Any thoughts on this one?

**Comment by Schweinepriester** [comment, marked as answer]
Nope, the entries mentioned on the badge just become "inaccessible". So your achievement will stay.
Let me quickly find an example :)

**Reply by Schweinepriester** [reply]
Here we go, it will look like that then:

https://github.com/thenithinbalaji?achievement=quickdraw&tab=achievements
![image](https://user-images.githubusercontent.com/2644614/179353066-44be6d8a-d00d-4994-9359-59bb0bce5e50.png)

**Reply by OlaPietka** [reply]
Thank you! :)

**Comment by ulp1an** [comment]
I don't think so

**Comment by quartesa37** [comment]
#6
